server/apps/archive/archive_spike.py

```
@@ -69,9 +69,21 @@ class ArchiveSpikeService(BaseService):
 
     def on_update(self, updates, original):
         updates[ITEM_OPERATION] = ITEM_SPIKE
+        self._validate_item(original)
         self._validate_take(original)
         self._update_rewrite(original)
 
+    def _validate_item(self, original):
+        '''
+        Raises an exception if the item is linked in a non-take package, the idea being that you don't whant to
+        inadvertently remove thing from packages, this force that to be done as a conscious action.
+        :param original:
+        :return: An excetption or nothing
```

Better to use `raises` instead of `return`